concent_api/core/models.py

```diff
@@ -365,7 +365,6 @@ class SubtaskState(ChoiceEnum):
 
     class Meta:
         unique_together = (
-            ('requestor', 'task_id'),
             ('requestor', 'subtask_id'),
```

Comment:
shouldn't we remove this as well? and have 
```python
unique_together = (
    ('task_id', 'subtask_id')
)
```
instead?

Reply:
ok, so it would seem we need `('requestor', 'task_id')` **and** `('task_id', 'subtask_id')`?

Reply:
It might work like this, but I think this change will result in fixing some decent amount of tests, so if you want to do this add a separate issue for this. This issue is only about removing one of the constrains.

Reply:
@cameel, your thoughts?

Reply:
@cameel @dybi I think it requires very small amount of work, it would be only adding constraint, migration and few extra tests. I doubt that more than several existing tests will fail, as they already use unique `subtask_id`.

Reply:
Created https://github.com/golemfactory/concent/issues/1096.